platform/kube/controller_test.go

```diff
@@ -102,10 +102,15 @@ func TestServices(t *testing.T) {
 	eventually(func() bool {
 		out := sds.Services()
 		glog.Info("Services: %#v", out)
-		return len(out) == 1 &&
-			out[0].Hostname == hostname &&
-			len(out[0].Ports) == 1 &&
-			out[0].Ports[0].Protocol == model.ProtocolHTTP
+
+		for _, item := range out {
```

Comment:
does not check for out = 1 anymore? Probably related to the fact that test listens on all namespaces, which it shouldn't.